watertap/property_models/anaerobic_digestion/adm1_reactions.py

```diff
@@ -1290,6 +1295,72 @@ def _rxn_rate(self):
             units=pyo.units.kmol / pyo.units.m**3,
         )
 
+        def Dissociation_rule(self, t):
+            return (
+                self.KW
+                == (
+                    1e-14
```

Comment:
Same comment here about defining some of these constants as parameters so that it's more clear where they come from/what they represent.